resources/views/partials/bootstrap-table.blade.php

```diff
@@ -1,25 +1,16 @@
-<script src="{{ asset('js/bootstrap-table.min.js') }}"></script>
-<script src="{{ asset('js/extensions/mobile/bootstrap-table-mobile.js') }}"></script>
-<script src="{{ asset('js/extensions/export/bootstrap-table-export.js?v=1') }}"></script>
-<script src="{{ asset('js/extensions/export/jquery.base64.js') }}"></script>
-<script src="{{ asset('js/FileSaver.min.js') }}"></script>
-<script src="{{ asset('js/xlsx.core.min.js') }}"></script>
-<script src="{{ asset('js/jspdf.min.js') }}"></script>
-<script src="{{ asset('js/jspdf.plugin.autotable.js') }}"></script>
-<script src="{{ asset('js/extensions/export/tableExport.min.js') }}"></script>
+@push('css')
```

Comment:
Wouldn't this push these into the top of the page for loading, and potentially increase load times for pages that don't even use the tables? (This is an honest question - I haven't used stack or push before.)

Reply:
Actually, no, I suppose it wouldn't, since that would only be pushed if the bootstrap-tables partial is included.

Reply:
@snipe That is correct. Using the stack and push, it's easy to make sure the js or css is loading in the correct order.

Reply:
That's really cool - I wish I had known about them, like, oh, 3 years ago lol